doc/flame/components.md

```diff
@@ -7,10 +7,27 @@ This diagram might look intimidating, but don't worry, it is not as complex as i
 
 ## Component
 
-All components inherit from the abstract class `Component`.
+All components inherit from the abstract class `Component` and all components can have other
+`Component`s as children. This is the base of what we call the Flame Component System, or FCS for
+short.
 
-If you want to skip reading about abstract classes you can jump directly to
-[](#positioncomponent).
+Children can be added either with the `add(Component c)` method or directly in the constructor.
+
+Example:
+
+```dart
+void main() {
+  final component1 = Component(children: [Component(), Component()]);
+  final component2 = Component();
+  component2.add(Component());
+  component2.addAll([Component(), Component()]);
+}
+```
+
+`Component` here could of course be any type of `Component`.
+
+If you want to skip reading about the abstract `Component` class you can jump directly to the most
+common `Component` basis, the [`PositionComponent`](#positioncomponent).
```

Comment:
```suggestion 
If you want to skip the abstract `Component` class and go directly to the one that most games will use, see the [`PositionComponent`](#positioncomponent).
```

The sentence felt a bit "bloated" if that makes sense? Didnt get to the point quickly enough for me haha (yes i would skip directly to the position component)

Reply:
This isn't very clear either I think, since you most of the time don't use the `PositionComponent` directly in your game.
I'll write up another a suggestion after dinner. :)

Reply:
I removed the sentence completely, it doesn't really make sense to skip the lifecycle to jump directly to `PositionComponent` imho. And I made the component lifecycle into a section instead.